### DISCUSSION TITLE
Docker file ?

### DISCUSSION THREAD

**Discussion by rezzalex** [Motion-Project/motionplus#40] [Q&A] [answered]
Hello,

would it be possible to have a docker file  or, even better, a docker image ?

I saw something in this thread, but not sure... : https://github.com/Motion-Project/motionplus/discussions/29

thanks a lot

**Comment by Mr-Dave** [comment, marked as answer]
That wouldn't be implemented by Motion-Project.  You would need to do that yourself.

**Comment by v0r0n0v** [comment]
Hi, I managed to get it running
motionplus in Docker, although this is a quick solution (the size of the image leaves much to be desired :D  ), maybe it will suit someone.

https://github.com/v0r0n0v/motionplus-docker
